src/components/companies/NewCompany.tsx

```diff
@@ -0,0 +1,186 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function NewCompany() {
+  const navigate = useNavigate();
+
+  const [name, setName] = useState<string>('');
+  const [website, setWebsite] = useState<string>('');
+  const [streetAddress, setStreetAddress] = useState<string>('');
+  const [city, setCity] = useState<string>('');
+  const [state, setState] = useState<string>('');
+  const [zipCode, setZipCode] = useState<string>('');
+  const [notes, setNotes] = useState<string>('');
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const newCompany = {
+      name,
+      website,
+      street_address: streetAddress,
+      city,
+      state,
+      zip_code: zipCode,
+      notes
+    };
+
+    try {
+      const token =
+        "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoyLCJyb2xlcyI6WyJ1c2VyIl0sImV4cCI6MTczNDE4ODQ3Nn0.h2YmilmQWZ8CmopONeNZNZpiHIqtdR7M2Siw7jDISaU";
```

Comment:
Same as previous, I'm not sure we want to have an example of what the token is in our code.